src/style.css

```diff
@@ -0,0 +1,181 @@
+  text-decoration: none;
+}
+
+.container {
+  display: flex;
+  max-width: 1200px;
+  min-height: 100vh;
+  margin: 0 auto;
+}
+
+.stars {
+  display: flex;
+  align-items: center;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background: url(images/star.svg) center no-repeat;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--1 .stars__star:first-child,
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--5 > * {
+  background: url(images/star-active.svg) center no-repeat;
+}
+
+.card {
+  max-width: 200px;
+  padding: 32px 16px 16px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.container__card {
+  margin: auto;
+}
+
+.card__link {
+  display: block;
+  outline-color: var(--secondary-color);
+}
+
+.card__img {
+  display: block;
+  max-width: 100%;
+  padding: 0 3px;
+  margin-bottom: 40px;
+  text-align: center;
+  outline-color: var(--secondary-color);
+}
+
+.card__title {
+  display: block;
+  margin-bottom: 4px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--main-text-color);
+  outline-color: var(--secondary-color);
+
+}
+
+.card__product-code {
+  margin-bottom: 16px;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 140%;
+  color: #616070;
+}
+
+.card__stars {
+  display: flex;
+  align-items: flex-end;
+  justify-content: space-between;
+  margin-bottom: 24px;
+}
+
+.card__stars-review {
+  position: relative;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--main-text-color);
+  transition: 0.5s color;
+  outline-color: var(--secondary-color);
+
+}
+
+.card__stars-review:hover {
+  color: var(--secondary-color);
+}
+
+.card__stars-review:hover::after {
+  transform: scaleX(1);
+  transform-origin: left;
+
+}
+
+.card__stars-review:after {
+  content: "";
+  position: absolute;
+  height: 1px;
+  width: 100%;
+  bottom: 0;
+  left: 0;
+  background-color: var(--secondary-color);
+  transform: scaleX(0);
+  transform-origin: right;
+  transition: 0.5s transform;
+}
+
+.card__prices {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+}
+
+.card__price {
+  color: #616070;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 150%;
+}
+
+.card__value {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 112%;
+}
+
+.card__button {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 166px;
+  height: 40px;
+  text-align: center;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 114%;
+  color: #fff;
+  background: var(--secondary-color);
+  border-radius: 5px;
+  text-transform: uppercase;
+  border: 1px solid transparent;
+  transition: 1s all;
+  outline-color: var(--main-text-color);
+}
+
+.card__button:hover {
+  color: var(--secondary-color);
+  background: none;
+  border-color: var(--secondary-color);
+}
```

Comment:
You need create seperate file for each BEM element https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md (point 11)